statsmodels/tsa/stattools.py

```diff
@@ -405,27 +449,10 @@ def acf(x, unbiased=False, nlags=40, qstat=False, fft=False, alpha=None):
 
     If unbiased is true, the denominator for the autocovariance is adjusted
     but the autocorrelation is not an unbiased estimtor.
-    '''
-    nobs = len(x)
-    d = nobs  # changes if unbiased
-    if not fft:
-        avf = acovf(x, unbiased=unbiased, demean=True)
-        #acf = np.take(avf/avf[0], range(1,nlags+1))
-        acf = avf[:nlags + 1] / avf[0]
-    else:
-        x = np.squeeze(np.asarray(x))
-        #JP: move to acovf
-        x0 = x - x.mean()
-        # ensure that we always use a power of 2 or 3 for zero-padding,
-        # this way we'll ensure O(n log n) runtime of the fft.
-        n = _next_regular(2 * nobs + 1)
```

Comment:
before this gets lost: `_next_regular` should be used in acovf
(I don't know why it doesn't show up in the original PR)